src/rpc/util.cpp

```diff
@@ -139,6 +139,8 @@ RPCErrorCode RPCErrorFromTransactionError(TransactionError terr)
         case TRANSACTION_ERR_P2P_DISABLED:
             return RPC_CLIENT_P2P_DISABLED;
         case TRANSACTION_ERR_INVALID_PSBT:
+        case TRANSACTION_ERR_PSBT_MISMATCH:
```

Comment:
This changes the error code for TRANSACTION_ERR_INVALID_PSBT from RPC_DESERIALIZATION_ERROR to RPC_INVALID_PARAMETER. Is that intentional?

Reply:
Yes, because there are two cases where we return an RPC error for a PSBT being semantically invalid, and we were returning DESERIALIZATION_ERROR in one case (when the invalid PSBT was given as a parameter) and INVALID_PARAMETER in the other (when the invalid PSBT was the result of a merge), and the I though the latter seemed more reasonable. My sense was that DESERIALIZATION_ERROR is only meant for syntactically invalid arguments? I'd be willing to change it back though. It would be easier if the two cases were treated the same.

Reply:
I don't have any strong opinion on the RPC coded. I just wanted to make sure this was intentional (as it could easily have been a copypaste error or so).